test/Elastic.Apm.Tests/Mocks/TestLogger.cs

```diff
@@ -17,6 +17,6 @@ internal class TestLogger : ConsoleLogger
 		public TestLogger(LogLevel level, StringWriter writer) : base(level, writer, writer) => _writer = writer;
 
 		public IReadOnlyList<string> Lines =>
-			_writer.GetStringBuilder().ToString().Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries).ToList();
+			_writer.GetStringBuilder().ToString().Split(Environment.NewLine.ToCharArray(), StringSplitOptions.RemoveEmptyEntries).ToList();
```

Comment:
The `Split` method with the previous parameter list does not exist on `net461`, that's why I added the `.ToCharArray()` here.